cpu/efm32/cpu.c

```diff
@@ -83,6 +83,10 @@ static void dcdc_init(void)
  */
 static void clk_init(void)
 {
+    /* disable LFRCO comparator chopping and dynamic element matching
+     * else LFRCO has too much jitter for LEUART > 1800 baud */
+    CMU->LFRCOCTRL &= ~(CMU_LFRCOCTRL_ENCHOP | CMU_LFRCOCTRL_ENDEM);
```

Comment:
looks like `CMU_LFRCOCTRL_ENDEM` is not always available.

Maybe wrap this in an `#ifdef CMU_LFRCOCTRL_ENDEM`?